lib/commands/install.ts

```diff
@@ -0,0 +1,72 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import path = require("path");
+
+export class InstallCommand implements ICommand {
+	constructor(private $fs: IFileSystem,
+		private $errors: IErrors,
+		private $logger: ILogger,
+		private $options: IOptions,
+		private $injector: IInjector,
+		private $staticConfig: IStaticConfig) { }
+		
+	allowedParameters: ICommandParameter[] = [];
+	
+	execute(args: string[]): IFuture<void> {
+		return (() => {
+			let projectFilePath = this.getProjectFilePath(args[0]);
+			let projectData = this.getProjectData(projectFilePath).wait();
+			let projectName = projectData.id.split(".")[2];
```

Comment:
Consider validating projectName, because package.json is under user control and can have invalid content.